docs/native/src/overview.rst

```diff
@@ -1,5 +1,76 @@
+
+
+Integration process
+-------------------
+
+You can use the Web Application Open Platform Interface (WOPI) protocol to integrate Office for iOS with your application. The WOPI protocol enables Office for iOS to access and change files that are stored in your service.
+
+To integrate your application with Office for iOS, you need to do the following:
+ #. Become a member of the Office 365 - Cloud Storage Partner Program. Currently integration with Office for iOS using WOPI is only available to cloud storage partners. You can learn more about the program, as well as how to apply, at `Office 365 Cloud Storage Partner Program. <http://dev.office.com/programs/officecloudstorage>`_
+ #. Provide the required on-boarding information as described in the section titled :ref:`Onboarding information <onboarding>`.
+ #. Obtain your ProviderId and app store URLs from Microsoft. The app store URLs are the URLs you should use to launch Office on a platform's app store. 
+ #. Add your domain to the WOPI domain :ref:`allow list <allow list>`. This is required even if you do not integrate with Office Online so you can verify your WOPI integration with the :ref:`validator app <Validator app>`.
+ #. Implement the WOPI protocol - a set of REST endpoints that expose information about the documents that you want to view or edit in Office for iOS. The set of WOPI operations that must be supported is described in the section titled :ref:`WOPI implementation requirements for Office for iOS integration <requirements>`.
+ #. Implement required changes to your app, including all applicable promotion requirements, as well as :ref:`opening files from your app into Office <OpenFilesiOSOffice>`. Contact your Office for iOS integration contacts to receive directions to run Office for iOS in test mode, to enable testing. 
+ #. Run the :ref:`validator app <Validator app>` and fix any issues until the validator reports a 100% pass rate. 
+ #. Complete the manual testing outlined in the :ref:`Manual Validation section <Manual Validation>` to verify the integration works as expected. 
+ #. Once all manual test cases have a 100% pass rate, send your :download:`Manual Validation document <../testing/PartnerTestCasesReport.docx>`to the Office for iOS integration contacts so they can schedule manual testing. In addition to the Manual Validation results, please also include directions to run your validator, which should be 100% PASSED or SKIPPED. We will need accounts and access to your production service to complete final testing. 
+ #. Once the Office for iOS team completes their manual testing, they will work with you to schedule your launch date. 
+	
+Authentication
+~~~~~~~~~~~~~~
+
+Authentication is handled by passing Office for iOS an access token that you generate from a Bootstrapper URL. Assign this token a reasonable expiration date. Also, we recommend that tokens be valid for a single user against a single file to help mitigate the risk of token leaks.
```

Comment:
There's an authentication/security section which are both in progress. I'll update this section to link to those once they're done.